src/palace/manager/api/overdrive/api.py

```diff
@@ -386,7 +231,7 @@ def child_settings_class(cls) -> type[OverdriveChildSettings]:
 
     @classmethod
     def label(cls) -> str:
-        return DataSourceConstants.OVERDRIVE
+        return OVERDRIVE_LABEL
```

Comment:
This had to get moved into the `OVERDRIVE_LABEL` constant outside of the class in order to break some import cycles.